zilliqa/tests/it/unreliable.rs

```diff
@@ -0,0 +1,35 @@
+use crate::Network;
+
+#[zilliqa_macros::test]
+async fn blocks_are_produced_while_a_node_restarts(mut network: Network) {
+    let restarted_node = network.random_index();
+    let wallet = network.wallet_of_node(restarted_node).await;
+
+    // Select a wallet connected to a different node, so we can query the network when the first node is disconnected.
+    let other_wallet = loop {
+        let i = network.random_index();
+        if i != restarted_node {
+            break i;
+        }
+    };
+    let other_wallet = network.wallet_of_node(other_wallet).await;
+
+    // Produce a few blocks to start with. Enough for everyone to join the consensus committee.
+    // TODO(#721): Once the committee is visible in the API, we can avoid waiting as long.
+    network.run_until_block(&wallet, 8.into(), 400).await;
```

Comment:
Is 8 blocks actually necessary in your experience? That seems like quite a long time to wait just for consensus to stabilise on genesis.

Reply:
The test would failllll ona  few random test cases with a lower block number. What was happening was that only a single node was in the committee when we passed this point and that was also the node that got disconnect. This obviously meant no blocks were produced while the node was disconnected.

So I just raised this to a number that makes it very likely there will be at least 2 nodes in the committee.

I raised #721 to improve this though - We should really be doing "run until the committee contains at least 2 nodes".